Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [features]
 default = ["debug", "cli"]
+bin = []
```

Comment:
don't see this feature being used inside the code. am I missing it?

Reply:
some of the constructs in ipa_bench folder i.e test_mpc, helper and ipa_bench are using it. CC: @taikiy

Reply:
can you folks point me to one of those constructs? I don't seem to be able to find it